src/kakarot/model.cairo

```diff
@@ -25,6 +25,14 @@ namespace model {
         calldata: felt*,
         calldata_len: felt,
         value: felt,
+        block_context: BlockContext*,
+    }
+
+    struct BlockContext {
+        block_number_len: felt,
+        block_number: felt*,
+        block_hash_len: felt,
+        block_hash: felt*,
```

Comment:
Adding pointer to BlockContext containing arrays of block number and hash to CallContext